x-pack/plugins/lists/server/services/exception_lists/exception_list_client.mock.ts

```diff
@@ -47,11 +63,189 @@ export class ExceptionListClientMock extends ExceptionListClient {
+  const {
+    comments,
+    created_at,
+    created_by,
+    description,
+    entries,
+    item_id,
+    list_id,
+    meta,
+    name,
+    os_types,
+    tags,
+    tie_breaker_id,
+    type,
+    updated_by,
+  } = getExceptionListItemSchemaMock();
+  /* eslint-enable @typescript-eslint/naming-convention */
+
+  const soSchema: ExceptionListSoSchema = {
+    comments,
+    created_at,
+    created_by,
+    description,
+    entries,
+    immutable: undefined,
+    item_id,
+    list_id,
+    list_type: 'item',
+    meta,
+    name,
+    os_types,
+    tags,
+    tie_breaker_id,
+    type,
+    updated_by,
+    version: undefined,
+    ...overrides,
+  };
+
+  return soSchema;
+};
+
+/**
+ * Returns a Saved Object with the `ExceptionListSoSchema` as the attributes
+ * @param attributesOverrides
+ * @param savedObjectOverrides
+ */
+export const getExceptionListItemSavedObject = (
+  attributesOverrides: Partial<ExceptionListSoSchema> = {},
+  savedObjectOverrides: Partial<Omit<SavedObject, 'attributes'>> = {}
+): SavedObject<ExceptionListSoSchema> => {
+  return {
+    attributes: getExceptionListSoSchemaMock(attributesOverrides),
+    coreMigrationVersion: undefined,
+    error: undefined,
+    id: ID,
+    migrationVersion: undefined,
+    namespaces: undefined,
+    originId: undefined,
+    references: [],
+    type: getSavedObjectType({ namespaceType: 'agnostic' }),
+    updated_at: DATE_NOW,
+    version: _VERSION,
+    ...savedObjectOverrides,
+  };
+};
+
+/**
+ * Returns a saved objects client mock that includes method mocks to handle working with the exceptions list client.
+ * @param [soClient] can be provided on input and its methods will be mocked for exceptions list only and will preserve existing ones for other types
+ */
+export const getExceptionListSavedObjectClientMock = (
+  soClient: ReturnType<typeof savedObjectsClientMock.create> = savedObjectsClientMock.create()
+): ReturnType<typeof savedObjectsClientMock.create> => {
+  // mock `.create()`
+  const origCreateMock = soClient.create.getMockImplementation();
+  // @ts-expect-error
+  soClient.create.mockImplementation(async (...args) => {
+    const [type, attributes] = args;
+
+    if (isExceptionsListSavedObjectType(type)) {
+      return getExceptionListItemSavedObject(attributes as ExceptionListSoSchema, { type });
+    }
+
+    return origCreateMock ? origCreateMock(...args) : undefined;
+  });
+
+  // Mock `.update()`
+  const origUpdateMock = soClient.update.getMockImplementation();
+  soClient.update.mockImplementation(
+    // @ts-expect-error
+    async (...args) => {
+      const [type, id, attributes, { version } = { version: undefined }] = args;
+
+      if (isExceptionsListSavedObjectType(type)) {
+        return getExceptionListItemSavedObject(attributes as ExceptionListSoSchema, {
+          id,
+          type,
+          version: version ?? _VERSION,
+        });
+      }
+
+      return origUpdateMock ? origUpdateMock(...args) : undefined;
```

Comment:
See comments ([above](https://github.com/elastic/kibana/pull/121324/files#r778306441)), but change this to just:

```ts
origUpdateMock(...args);
```